src/TorSharp/TorSharpProxy.cs

```diff
@@ -109,7 +109,32 @@ public async Task GetNewIdentityAsync()
                 await client.QuitAsync().ConfigureAwait(false);
             }
         }
-
+        /// <summary>
+        /// Get traffic read data.
+        /// </summary>
+        public async Task<String> GetTrafficReadAsync()
```

Comment:
I'm wondering how many of the control client methods should also be lifted to the `TorSharpProxy` class. If we lift all of them then I'm worried the `TorSharpProxy` type would become bloated over time.